packages/toggle/stories/Toggle.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { withReadme } from "storybook-readme";
+import { Toggle } from "../../index";
+
+const readme = require("../README.md");
+
+const primary = () => <div>primary component</div>;
+const secondary = () => <div>secondary component</div>;
+
+storiesOf("Toggle", module)
+  .addDecorator(withReadme([readme]))
+  .addWithInfo("string", () => (
+    <Toggle primaryContent="Hello" secondaryContent="Bye" />
```

Comment:
Pass a `tabIndex` to the story - it defaults to -1

Not part of this PR, but something to consider for the Clickable component:
`:focus { outline: 0; }` hides any visual indication that the element has focus. Where might I file a ticket to improve the keyboard navigation for that component?

Reply:
I will remove in this PR because it's pretty small overhead